Library/Formula/uwsgi.rb

```diff
@@ -45,6 +45,10 @@ class Uwsgi < Formula
   depends_on "v8" => :optional
   depends_on "zeromq" => :optional
 
+  # Fix build on case-sensitive filesystems
+  # https://github.com/Homebrew/homebrew/issues/45560
+  patch :DATA
```

Comment:
Switched it to `inreplace` for better readability. Still builds on case-sensitive HFS.